packages/components/src/FormField/FormField.tsx

```diff
@@ -250,3 +245,13 @@ function setAutocomplete(
 
   return autocompleteSetting;
 }
+
+function extractErrorMessage(errors: FieldErrors, fieldName: string): string {
```

Comment:
It was suggested to me by @JeffMarvin that I check out what was done for this in `components-native` and I found [this](https://github.com/GetJobber/atlantis/blob/f228ad76309d57f68e2fac220a000bfa7927fc81/packages/components-native/src/hooks/useFormController.ts#L52).

It appears that that effort was less flexible, handling only 1 or 3 'partls', with a note to make it more flexible. This implementation here will cover any number of 'parts', and so is more flexible.

Reply:
Its interesting that this was required, but in reflecting upon it, its all due to not leveraging `useController` and instead leveraging `useFormContext`. Ideally by leveraging the controller you get updates automatically. Seems like a good compromise though.

Reply:
You can avoid all of this by just getting the error from the Controller from its `fieldState` attribute in the render function. It'll give you the targetted error directly.

Reply:
Changed in 08fca431d3fe53b10b51f1d0ff37a7015db888b8
So much nicer! 
I always hold my nose whenever I reach for lodash, and now I don't have to!

Reply:
Should we change mobile to have the same logic too? We can delete ~10 lines by just making it
```
  const { field, fieldState: { error } } = useController({
```

Reply:
I'm open to that, but I wouldn't do it as part of this PR. I feel like the QA flow would be different for any changes on the mobile side. But I'm new to Atlantis, so maybe someone else would frame it differently? 🤷

Reply:
I was getting multiple test failures on JO with the `useController` implementation. The issue, I think, was that we were double-registering each component. I moved the error message logic inside the existing `Controller` and removed the `useController` in 834f7e49ed1da741a05cc83b376274e35036df2c